### ISSUE TITLE
Cannot render Lit components with `v-for` directives on them

### ISSUE THREAD

**Issue by steveworkman** [prashantpalikhe/nuxt-ssr-lit#33]
**Describe the bug**

When you have a `v-for` directive on a lit element, it does not SSR the contents.

This does not render:
```html
<my-button :label="a" v-for="a in ['one', 'two', 'three']"></my-button>
```

This does render
```html
<template v-for="a in ['one', 'two', 'three']">
 <my-button :label="a"></my-button>
</template>
```

**Additional context**

This produces VNodes that have a tagName of `Symbol(Fragment)`. Trying to pass this to the `h` function results in further issues where `children` are `undefined` deep in the server renderer. 

**Diagnosis**

I _think_ that the `autoLitWrapper` is going to wrap the source file, which will then only wrap the virtual element that is created in the `v-for`, so the children are not wrapped correctly

**Comment by prashantpalikhe** [comment]
This is going to be a bit difficult to implement. Maybe we should just add using the wrapping template for `v-for` in the caveat section for now. In the future, we can look into better code generation while auto wrapping, which takes the `v-for` into account.

**Comment by prashantpalikhe** [comment]
Fixed by #53
